src/main/java/ro/msg/learning/shop/model/Address.java

```diff
@@ -0,0 +1,26 @@
+package ro.msg.learning.shop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.*;
```

Comment:
Suggestion: Please use explicit imports, do not import all the things from lombok with *. You don't need all the annotations from lombok. Please follow the same approach everywhere in the code